docs/coordinates/satellites.rst

```diff
@@ -0,0 +1,105 @@
+code, designed to be compatible with the TLE sets provided by the United
+States Department of Defense, which are available from a source like
+`Celestrak <http://celestrak.com/>`_.
+
+The output coordinate frame of the SGP4 model is the True Equator, Mean Equinox
+frame (TEME), which is one of the frames built-in to `astropy.coordinates`.
+TEME is an Earth-centered inertial frame (i.e it does not rotate with respect
+to the stars). Several definitions exist; astropy uses the implementation described
+in `Vallado et al (2006) <https://celestrak.com/publications/AIAA/2006-6753/AIAA-2006-6753-Rev2.pdf>`_.
+
+Finding ``TEME`` coordinates from TLE data
+==========================================
+
+There is currently no support in `astropy.coordinates` for computing satellite orbits
+from TLE orbital element sets. Full support for handling TLE files is available in
+the `Skyfield <http://rhodesmill.org/skyfield/>`_ library, but some advice for dealing
+with satellite data in astropy is below.
+
+..
+  EXAMPLE START
+  Using sgp4 to get a TEME coordinate
+
+You will need some external library to compute the position and velocity of the satellite from the
+TLE orbital elements. The `sgp4 <https://pypi.org/project/sgp4/>`_ library can do this. An example
+using this library to find the  `~astropy.coordinates.TEME` coordinates of a satellite is::
+
+.. doctest-requires:: sgp4
+
+        >>> from sgp4.api import Satrec
+        >>> from sgp4.api import SGP4_ERRORS
+        >>> s = '1 25544U 98067A   19343.69339541  .00001764  00000-0  38792-4 0  9991'
+        >>> t = '2 25544  51.6439 211.2001 0007417  17.6667  85.6398 15.50103472202482'
+        >>> satellite = Satrec.twoline2rv(s, t)
+
+The ``satellite`` object has a method, ``satellite.sgp4``, that will try to compute the TEME position
+and velocity at a given time::
+
+.. doctest-requires:: sgp4
+
+        >>> from astropy.time import Time
+        >>> t = Time(2458827.362605, format='jd')
+        >>> error_code, teme_p, teme_v = satellite.sgp4(t.jd1, t.jd2)  # in km and km/s
+        >>> if error_code != 0:
+        ...     raise RuntimeError(SGP4_ERRORS[error_code])
+
+Now we have the position and velocity in kilometers and kilometers per second, we can create a
+position in the `~astropy.coordinates.TEME` reference frame::
+
+.. doctest-requires:: sgp4
+
+        >>> from astropy.coordinates import TEME, CartesianDifferential, CartesianRepresentation
+        >>> from astropy import units as u
+        >>> teme_p = CartesianRepresentation(teme_p*u.km)
+        >>> teme_v = CartesianDifferential(teme_v*u.km/u.s)
+        >>> teme = TEME(teme_p.with_differentials(teme_v), obstime=t)
+
+Note how we are careful to set the observed time of the `~astropy.coordinates.TEME` frame to
+the time at which we calculated satellite position.
+
+Transforming ``TEME`` to other coordinate systems
+==================================================
+
```

Comment:
```suggestion

.. EXAMPLE START: Transforming TEME to Other Coordinate Systems

```

Reply:
I sorted out the example in a separate commit, rather than have two different commits.